### PR TITLE
Add frontend and backend

### PR DESCRIPTION
**Pull request by Shalinit3**
Added frontend and backend to the source. Also updated the Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
...
+
+### Ghost Modes:
+- Chase Mode
+- Scatter Mode
+- Frightened Mode
+
+
+### Tech Stack: 
+
+- Project management: GitHub issues
```
Comment (nimish-gupta) [inline README.md]: You can also use github projects

File: README.md [hunk 2]
```diff
@@ -1 +1,48 @@
...
+- Game Rendering: Canvas
+- Deployment: Heroku (Backend), Netlify(Frontend)
+- Authentication: Token based (JWT) / Passport.js
+- Database: Mongodb (mlab)
+- CI/CD: CircleCI
+
+
+### Database Schema:
```
Comment (nimish-gupta) [inline README.md]: is this the schema for the entire app or just for the User model?

**Review by nimish-gupta — COMMENTED** [review]
Good PR, for server-side, have you used any boilerplate?

File: README.md (hunk 2 above)
Comment (Shalinit3) [reply README.md]: Just for user model.

File: README.md (hunk 1 above)
Comment (Shalinit3) [reply README.md]: Alright will do the change

File: README.md (hunk 2 above)
Comment (nimish-gupta) [reply README.md]: Please update the Readme, also schema info should not be in the top Readme, it must be with `server/db/README.md`

**Comment by Shalinit3** [comment]
No, did not use any boilerplate.

**Comment by Shalinit3** [comment]
Updated the Readme

**Review by nimish-gupta — APPROVED** [review]
LGTM (looks good to merge)

**Pull request merged**